ConcernsCaseWork/Service.TRAMS/Decision/DecisionService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Service.TRAMS.Base;
+using Service.TRAMS.Helpers;
+using Service.TRAMS.Nti;
+using Service.TRAMS.Records;
+using System;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Service.TRAMS.Decision
+{
+	public class DecisionService :  AbstractService, IDecisionService

```

Comment:
Going to approve, but going to create a bug on the board for this. 
The base type for services has a method in it for posting, to avoid us having to re-write almost the same code over and over to call GET/POST/PUT etc endpoints. If we find a case that's not covered then that might be an exception, but probably more likely the base class needs to support new functionality.